src/const/props.ts

```diff
@@ -275,4 +275,22 @@ export interface GridProps extends GenericProps {
    * @see https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row-gap
    */
   gutterRow?: Numeric
+  /**
+   * Size of implicitly created rows
+   * @css grid-auto-rows
+   * @see https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-rows
+   */
+  autoRows?: string | Numeric
```

Comment:
Sorry for confusion, but `Numeric` is already a union for `number` and `string` :)

https://github.com/kettanaito/atomic-layout/blob/0ccca378e19c2793ba0b190773c295b10a428705/src/const/defaultOptions.ts#L13

That is because by "numeric" term I refer to the values that can be numbers (that are then suffixed with the default measurement unit), or strings, that serve as explicit number+unit (i.e. `10vh`).

May I please ask you to use just `Numeric`?

Reply:
Ah ok - I've made the change - also I renamed autoColumns to autoCols to fit with the rest of the API